ghost/core/core/server/services/i18n.js

```
@@ -3,7 +3,37 @@
 /** @type {import('i18next').i18n} */
 let i18nInstance;
 
-module.exports.init = function () {
+/**
+ * Loads translations for a specific locale
+ * @param {string} locale - The locale to load translations for
+ * @returns {Promise<void>} A promise that resolves when the translations are loaded
+ */
+async function loadLocale(locale) {
+    if (!i18nInstance) {
+        throw new Error('i18n not initialized');
```

ESLint

[error] Direct calls to new Error() are not allowed. Please use @tryghost/errors.

(ghost/ghost-custom/no-native-error)